Prevents gendef from putting annoying comment at beginning of .def file.

diff --git a/mingw-w64-tools/gendef/src/gendef.c b/mingw-w64-tools/gendef/src/gendef.c
index c25734c..08a2722 100644
--- a/mingw-w64-tools/gendef/src/gendef.c
+++ b/mingw-w64-tools/gendef/src/gendef.c
@@ -728,8 +728,6 @@ dump_def (void)
     fprintf(stderr," * failed to create %s ...\n",fnoutput);
     return;
   }
-  fprintf (fp,";\n; Definition file of %s\n; Automatic generated by gendef %s\n; written by Kai Tietz 2008\n",
-    fndllname, VERSION);
   if (has_atdecoration ())
     fprintf (fp,"; The def file has to be processed by --kill-at (-k) option of dlltool or ld\n");
   fprintf (fp,";\n");
